pkg/controller/catalog/util/converter.go

```diff
@@ -17,40 +17,53 @@ limitations under the License.
 package util
 
 import (
-	"encoding/json"
-
 	sbmodel "github.com/kubernetes-incubator/service-catalog/model/service_broker"
-	scmodel "github.com/kubernetes-incubator/service-catalog/model/service_controller"
+	"github.com/kubernetes-incubator/service-catalog/pkg/apis/servicecatalog"
 )
 
 // ConvertCredential converts a Credential object from the broker model to the
 // controller model.
-func ConvertCredential(in *sbmodel.Credential) (*scmodel.Credential, error) {
-	j, err := json.Marshal(in)
-	if err != nil {
-		return nil, err
-	}
+// func ConvertCredential(in *sbmodel.Credential) (*servicecatalog.BindingSpec, error) {
+// 	j, err := json.Marshal(in)
+// 	if err != nil {
+// 		return nil, err
+// 	}
+//
+// 	out := &scmodel.Credential{}
+// 	if err := json.Unmarshal([]byte(j), out); err != nil {
+// 		return nil, err
+// 	}
+//
+// 	return out, nil
+// }
 
-	out := &scmodel.Credential{}
-	if err := json.Unmarshal([]byte(j), out); err != nil {
-		return nil, err
+func convertServicePlans(plans []sbmodel.ServicePlan) []servicecatalog.ServicePlan {
+	ret := make([]servicecatalog.ServicePlan, len(plans))
+	for i, plan := range plans {
+		ret[i] = servicecatalog.ServicePlan{
+			Name:       plan.Name,
+			CFGUID:     plan.ID,
+			CFMetadata: plan.Metadata,
+			CFFree:     plan.Free,
+		}
 	}
-
-	return out, nil
+	return ret
 }
 
-// ConvertCatalog converts a Catalog object from the broker model to the
-// controller model.
-func ConvertCatalog(in *sbmodel.Catalog) (*scmodel.Catalog, error) {
-	j, err := json.Marshal(in)
-	if err != nil {
-		return nil, err
+// ConvertCatalog converts a service broker catalog into an array of ServiceClasses
```

Comment:
yea my comment wasn't so much about this PR as much as its about the original placement in here :-)  However, I wouldn't object to it being moved as part of this PR. Otherwise, if I remember, I'll do it in a follow-on PR.